tests/Query/Functions/Mysql/JsonPrettyTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Scienta\DoctrineJsonFunctions\Tests\Query\Functions\Mysql;
+
+use Scienta\DoctrineJsonFunctions\Tests\Query\MysqlTestCase;
+use Doctrine\ORM\Query\Expr;
+
+class JsonPrettyTest extends MysqlTestCase
+{
+    public function testJsonPretty()
+    {
+        $this->assertDqlProducesSql(
+            "SELECT JSON_PRETTY('\"null\"') from Scienta\DoctrineJsonFunctions\Tests\Entities\Blank b",
+            "SELECT JSON_PRETTY('\"null\"') AS sclr_0 FROM Blank b0_"
+        );
```

Comment:
Can you also add a test parsing more complex DQL?
SQL queries like `SELECT JSON_PRETTY(NULL);` and `SELECT JSON_PRETTY(CAST(CONCAT('[', '1,2,45,3', ']') AS JSON));` must also work.
Just testing `"null"` is a bit too small.